contrib/nydusify/cmd/nydusify.go

```diff
@@ -171,6 +176,14 @@ func getPrefetchPatterns(c *cli.Context) (string, error) {
 	return patterns, nil
 }
 
+type RegistryBackendConfig struct {
```

Comment:
Can we share the code with `check` subcommand implementation?

Reply:
Because the chaos of commit, i have submit a new pull-request #1317. And this quetion has been solve.

Reply:
@lihuahua123 Please don't submit a new one, just use `git rebase -i $commit^` to `edit` the commit and then force push by `git push -f $branch`.